MORKExample/MORKExample/ViewController.h

```diff
@@ -7,9 +7,8 @@
 //
 
 #import <UIKit/UIKit.h>
-#import <ResearchKit/ResearchKit.h>
 
-@interface ViewController : UIViewController <ORKTaskViewControllerDelegate>
-@property (strong, nonatomic) ORKTaskViewController *taskViewController;
```

Comment:
We can put this stuff in the class extension as no one outside our class is using it or needs to know about it.